src/strings/PowerFxResources.en-US.resx

```diff
@@ -4093,9 +4093,13 @@
     <value>The stated function return type '{0}' does not match the return type of the function body '{1}'.</value>
     <comment>This error message shows up when expected return type does not match with actual return type. The arguments '{0}' and '{1}' will be replaced with data types. For example, "The stated function return type 'Number' does not match the return type of the function body 'Table'"</comment>
   </data>
+  <data name="ErrUDF_TooManyParameters" xml:space="preserve">
+    <value>Function {0} has too many parameters.</value>
```

Comment:
>Function {0} has too many parameters. [](http://example.com/codeflow?start=11&length=37)

Can we add the limit number in error message? So that user can know about the limitation.